dfsclient/pom.xml

```diff
@@ -34,6 +34,7 @@
                 <value>org.hpccsystems.ws.client.TestResultNotifier</value>
               </property>
             </properties>
+            <excludesFile>../excluded-tests.txt</excludesFile>
```

Comment:
it seems ok to have a single excluded-tests file at the hpcc4j level, but I could also see an argument for dedicated excluded-tests files per sub project.
what are the main drawbacks/advantages of the current implementation?